packages/mobile/README.md

```diff
@@ -0,0 +1,31 @@
+# Jetpack Constants
+
+A method to detect mobile devices, formerly jetpack_is_mobile.
```

Comment:
Rename to `jetpack_device_type`, with `jetpack_is_mobile` as a compatibility layer.